docs/connectors/mongodb.asciidoc

```diff
@@ -240,94 +240,111 @@ Let's look at what a _create_/_read_ event value might look like for our `custom
         {
-          "field": "source",
           "type": "struct",
-          "name": "io.debezium.connector.mongo.Source",
-          "optional": false,
           "fields": [
             {
-              "field": "version",
               "type": "string",
-              "optional": false
+              "optional": false,
+              "field": "version"
             },
             {
-              "field": "name",
               "type": "string",
-              "optional": false
+              "optional": false,
+              "field": "connector"
             },
             {
-              "field": "rs",
               "type": "string",
-              "optional": false
+              "optional": false,
+              "field": "name"
+            },
+            {
+              "type": "int64",
+              "optional": false,
+              "field": "ts_ms"
+            },
+            {
+              "type": "boolean",
+              "optional": true,
+              "default": false,
+              "field": "snapshot"
             },
             {
-              "field": "ns",
               "type": "string",
-              "optional": false
+              "optional": false,
+              "field": "db"
             },
             {
-              "field": "sec",
-              "type": "int32",
+              "type": "string",
               "optional": false,
+              "field": "rs"
             },
             {
-              "field": "ord",
-              "type": "int32",
+              "type": "string",
               "optional": false,
+              "field": "collection"
             },
             {
-              "field": "h",
-              "type": "int64",
-              "optional": false
+              "type": "int32",
+              "optional": false,
+              "field": "ord"
             },
             {
-              "field": "initsync",
-              "type": "boolean",
-              "optional": true
+              "type": "int64",
+              "optional": true,
+              "field": "h"
             }
-          ]
+          ],
+          "optional": false,
+          "name": "io.debezium.connector.mongo.Source",
+          "field": "source"
+        },
+        {
+          "type": "string",
+          "optional": true,
+          "field": "op"
         },
         {
-          "field": "ts_ms",
           "type": "int64",
-          "optional": true
+          "optional": true,
+          "field": "ts_ms"
         }
-      ]
-    },
+      ],
+      "optional": false,
+      "name": "dbserver1.inventory.customers.Envelope"
+      },
     "payload": {
-      "op": "r",
-      "ts_ms": 1465491411815,
-      "after": "{\"_id\":1004,\"first_name\":\"Anne\",\"last_name\":\"Kretchmar\",\"email\":\"annek@noanswer.org\"}",
+      "after": "{\"_id\" : {\"$numberLong\" : \"1004\"},\"first_name\" : \"Anne\",\"last_name\" : \"Kretchmar\",\"email\" : \"annek@noanswer.org\"}",
+      "patch": null,
```

Comment:
Any idea why this shows up as `null` here now, @jpechane? I remember we discussed a similar question before, but what I don't remember is what the outcome of that was :)

Reply:
https://issues.jboss.org/browse/DBZ-984?focusedCommentId=13669063&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13669063

Reply:
Ah yes, thanks! I should bookmark that one ;) That said I'm wondering *why* the converter behaves that way; after all, there's a difference between not providing a value at all or explicitly setting it to null.